cdap-data-fabric/src/main/java/io/cdap/cdap/data2/metadata/lineage/field/FieldLineageTable.java

```diff
@@ -515,12 +516,9 @@ public List<EndPoint> getEndpoints(String namespaceId, ProgramReference programR
     return result;
   }
   private boolean isProgramRunIdMatchesProgramReferenceAndRunId(ProgramRunId programRunId,
```

Comment:
nit: this is a really long name, it's better to shorten it to something like `programRunMatches()`

Also should have a newline before the method.